Makefile

```diff
@@ -294,6 +294,16 @@ lint-fix: export TARGET := clojure
 lint-fix: ##@test Run code style checks and fix issues
 	clojure -Scp "$$CLASS_PATH" -m cljfmt.main fix src --indents indentation.edn
 
+## Start shadow-cljs in server mode for watching
+shadow-server: export TARGET := clojure
```

Comment:
hi @cammellos, where is `shadow-server` is used?

Reply:
I use it to keep it running so that further commands don't have to restart the server, in case I am testing things out, but I can remove it if necessary

Reply:
No, that's alright. Thanks for explaining.